### DISCUSSION TITLE
How to connect AWS RDS postgreSQL to django project

### DISCUSSION THREAD

**Discussion by Shinzo-ninja** [community/community#56575] [Programming Help]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

I have been working on a Django project and after making a lot progress in project now I am looking to migrate to postgre sql using AWS RDS so how can I connect django app to AWS RDS.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by hridesh-net** [comment]
install psycopg2 in your project by
`pip install psycopg2`

Then in the settings.py file search for DATABASES

and change  the settings according to the following:
`DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.postgresql',
        'NAME': '<database_name>',
        'USER': '<username>',
        'PASSWORD': '<password>',
        'HOST': '<RDS_endpoint>',
        'PORT': '5432',
    }
}
`

**Reply by Shinzo-ninja** [reply]
Thanks it worked
